MUXSDKStats/MUXSDKStats/MUXSDKPlayerBindingManager.m

```diff
@@ -59,10 +52,28 @@ - (void)dispatchDataEventForPlayerName:(NSString *)name playerData:(MUXSDKCustom
         [dataEvent setCustomerVideoData:customerVideoData];
     }
     if (customerPlayerData || customerVideoData) {
-       [MUXSDKCore dispatchEvent:dataEvent forPlayer:name];
+        dataEvent.videoChange = videoChange;
+        [MUXSDKCore dispatchEvent:dataEvent forPlayer:name];
     }
 }
 
+#pragma MARK - MUXSDKPlayDispatchDelegate
+
+- (void)playbackStartedForPlayer:(NSString *) name {
+    if (![self hasInitializedPlayerBinding: name]) {
+        NSLog(@"MUXSDK-WARNING - Detected SDK initialized after playback has started.");
+        [self newViewForPlayer:name];
+    }
+}
 
+- (void) videoChangedForPlayer:(NSString *_Nonnull) name {
+    MUXSDKPlayerBinding *binding = [self.viewControllers valueForKey:name];
+    if (binding != nil) {
+        [binding dispatchViewInit];
+        MUXSDKCustomerPlayerData *playerData = [self.customerPlayerDataStore playerDataForPlayerName:name];
+        MUXSDKCustomerVideoData *videoData = [self.customerVideoDataStore videoDataForPlayerName:name];
+        [self dispatchDataEventForPlayerName:name playerData:playerData videoData:videoData videoChange:YES];
```

Comment:
got it, this is where we use  what's in the store for fixing the videochange issue